src/lib.rs

```diff
@@ -78,6 +78,59 @@ impl PartialEq for general::__kernel_timespec {
 #[cfg(feature = "general")]
 impl Eq for general::__kernel_timespec {}
 
+mod cmsg_macros {
```

Comment:
I think you can fix the CI error by adding `#[cfg(feature = "general")]` on this `mod`.